xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -21,11 +21,10 @@
       <title>The feature described in this chapter is a <emphasis>Beta
       Feature</emphasis>.</title>
 
-      <para>The development of this feature is still in progress and under the
-      constant revision of the users involved in the beta program. This means
-      we can substantially change this feature throughout the process or even
-      discard it. So we can extend, modify and even delete the content of this
-      file at any time.</para>
+      <para>That means the development of this feature is still in progress and under
+      constant revision. We can substantially change this feature throughout the process
+      or even discard instances. Make sure to check back before reporting problems.
```

Comment:
The word "instances" confused me a bit. After looking up the dictionary I think you use it with the meaning of "cases". I wonder if that can be confusing for others too :thinking:

Reply:
@dmarcoux specific change was made by @hennevogel, maybe he can shed a bit of light here...

Reply:
It was a typo. @saraycp @dmarcoux take a look